src/Numerics/LinearAlgebra/Builder.cs

```diff
@@ -48,8 +48,12 @@ public override Matrix<double> Dense(DenseColumnMajorMatrixStorage<double> stora
             return new DenseMatrix(storage);
         }
 
-        public override Matrix<double> Sparse(SparseCompressedRowMatrixStorage<double> storage)
+        public override Matrix<double> Sparse(SparseCompressedRowMatrixStorage<double> storage, bool cleanup = false)
```

Comment:
Unfortunately adding an optional parameter is a breaking change, so I'll have to rework this slightly.

Reply:
Please rework the line. Thank you.

Reply:
I'll do - likely will have to become an overload.

Reply:
Ah, I did not realize the affected members are actually internal (since the class is internal). All is fine after all. Thanks again!